private/bufpkg/bufcheck/buflint/buflint_test.go

```diff
@@ -942,17 +931,104 @@ func TestCommentIgnoresCascadeOff(t *testing.T) {
 
 func TestCommentIgnoresCascadeOn(t *testing.T) {
 	t.Parallel()
+	testLintWithModifiers(t, "comment_ignores_cascade", func(config *bufconfig.Config) {
+		config.Lint.AllowCommentIgnores = true
+	}, nil, "")
+}
+
+func TestValidateRulesTypeDontMatch(t *testing.T) {
```

Comment:
The remainder of our test functions match the lint rule name, can you stick to this style?

Reply:
attempted to resolve in [1b2c71c](https://github.com/bufbuild/buf/pull/2377/commits/1b2c71cceeb54702ea58bc69970a1a434fb6410e). lmk if you want different names